ffmpeg-gpu/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM nvidia/cuda:8.0-cudnn5-devel
+MAINTAINER Ryan Versaw <ryan.versaw@hudl.com>
+
+RUN apt-get update -y
```

Comment:
Little nitpick - Docker says always put `apt-get update` and `apt-get install` in the same `RUN` - https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run